test/testGraphical.m

```diff
@@ -0,0 +1,47 @@
+function [ status ] = testGraphical( varargin )
+%TESTGRAPHICAL Runs the M2T test suite to produce graphical output
+%
+% This is quite a thin wrapper around testMatlab2tikz to run the test suite to
+% produce a PDF side-by-side report.
+%
+% Its allowed arguments are the same as those of testMatlab2tikz.
+%
+% Usage:
+%
+%     status = testGraphical(...) % gives programmatical access to the data
+%
+%     testGraphical(...); % automatically invokes makeLatexReport afterwards
+%
+% See also: testMatlab2tikz, testHeadless, makeLatexReport
+
+    cwd = initializeWorkingDirectory();
```

Comment:
I don't completely follow, to be honest. To do you mean move out almost all code outside of this main function? If so: I don't really bother with one way or another.